src/include/daos_srv/daos_engine.h

```diff
@@ -796,9 +796,11 @@ enum dss_media_error_type {
 
 void dss_init_state_set(enum dss_init_state state);
 
-/* Notify control-plane of a bio error. */
-int
-ds_notify_bio_error(int media_err_type, int tgt_id);
+/** Call module setup from drpc setup call handler. */
+int dss_module_setup_all(void);
+
+/** Notify control-plane of a bio error. */
+int ds_notify_bio_error(int media_err_type, int tgt_id);
```

Comment:
Note our coding style is and always has been the below.  We've just not been very good at sticking to it.  No need to re-push here but please consider this in future.
```suggestion
/** Call module setup from drpc setup call handler. */
int
dss_module_setup_all(void);

/** Notify control-plane of a bio error. */
int
ds_notify_bio_error(int media_err_type, int tgt_id);
```

Reply:
Thanks @ashleypittman , I was trying to keep consistency with the code in close proximity to the addition but I will stick to the style prescribed.